identity-account/tests/commands.rs

```diff
@@ -114,6 +117,29 @@ async fn test_create_identity_already_exists() -> Result<()> {
   Ok(())
 }
 
+#[tokio::test]
+async fn test_create_identity_from_secret_key() -> Result<()> {
```

Comment:
We may want to add equivalent tests to ensure we error on secret keys with invalid lengths, e.g. `let secret_key = [0_u8; 33].to_vec().into();`

Reply:
It's not going to error on that. `Vault::insert` takes an arbitrary vector of bytes. I'll add a check in the account.